Makefile

```diff
@@ -208,7 +208,7 @@ define start-cluster
 	@echo "**********************************"
 	@echo "Creating cluster"
 	@echo "**********************************"
-	@CLUSTER_NAME=uhs-test $(MAKE) create-cluster
+	$(MAKE) create-cluster
```

Comment:
Any specific reason for removing `@CLUSTER_NAME=uhs-test`? Creating a separate cluster for test seemed to be a good idea.

Reply:
Okay - reverted back in commit 70d0db9